examples/cloud_run_cloudevents/send_cloudevent.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from cloudevents.http import CloudEvent, to_structured_http
+import requests
+import json
+
+
+# Create event
+attributes = {
+    "Content-Type": "application/json",
+    "source": "from-galaxy-far-far-away",
+    "type": "cloudevent.greet.you"
+}
+data = {"name":"john"}
```

Comment:
Create a valid CloudEvent with the 4 required attributes.

Won't this fail with the CloudEvent SDK validation?

Reply:
Perhaps I should add further documentation. 

CloudEvent constructor has defaults for `ce-id` and `ce-specversion` so minimally you only need `ce-source` and `ce-type`. That being said, from_http expects full cloudevent required fields. In essence, when creating a brand new CloudEvent, it will generate new id's and default to specversion 1.0. When readding cloudevents, we expect everything else this was an invalid cloudevent.

Reply:
Added comment: 
```python
# CloudEvent constructor minimally requires a source and type field
```